util/regexp-assemble/data/932311.data

```diff
@@ -20,7 +20,7 @@
 
 ##! TDB: representing charset and language (if needed)
 
-##!^ (?is)\r\n\w{1,50}\b\ 
+##!^ (?is)\r\n\w{1,50}\b[ ]
```

Comment:
I would like the flags to be moved to the flag comment `##!+`. However, regexp-assemble currently explicitly only accepts the `i` flag, so the `s` flag would be dropped. IMO, that is OK, since `s` is implied by how the rule engine works. But I'm not 100% sure.

Reply:
Agreed. We might need to revisit this for _all_ engines (Coraza?)